test/drop_test/config.py

```diff
@@ -24,3 +24,84 @@ def __init__(self, ip: str, pubkey: bytes, privkey: bytes):
     ),
 }
 # fmt: on
+
+
+class File:
+    def __init__(self, size: int, id: str):
+        # in kilobytes
+        self.size = size
+        self.id = id
```

Comment:
I was thinking about reusing the `File` from `Action` but this actually is pretty nice. It's just a bit confusing as we have 3 classes called `File` in the test framework: Event, Action, Config. Maybe it makes sense to rename on of them?

Reply:
Changed `config.File` -> `config.TestFile`